wallet/db.c

```diff
@@ -200,6 +200,8 @@ char *dbmigrations[] = {
     "DELETE FROM peers WHERE id NOT IN (SELECT peer_id FROM channels);",
     /* The ONCHAIND_CHEATED/THEIR_UNILATERAL/OUR_UNILATERAL/MUTUAL are now one */
     "UPDATE channels SET STATE = 8 WHERE state > 8;",
+    /* Add bolt11 to invoices table*/
+    "ALTER TABLE invoices ADD bolt11 BLOB;",
```

Comment:
Maybe better as type `TEXT` since we store the BOLT11 invoice as encoded text.  I know SQLite3 does not really follow the indicated types very strictly, but it is good at least for documentation.